website/blog/unusual-builds-w-bytes.md

```diff
@@ -0,0 +1,217 @@
+---
+slug: unusual-builds-w-bytes
+title: "Unusual Builds with Bytes"
+description: "Fixing a download bug in Bazel Built without Bytes"
+authors: Son
+date: 2025-03-10:12:00:00
+image: /img/blog/troubleshooting.png
+tags: [bazel, engineering]
+---
+
+# Key Takeaways
+
+We discovered a bug in Bazel that was causing builds with `--remote_download_minimal` to download all output artifacts.
+
+A fix was submitted upstream to the Bazel repository and will be part of the Bazel 9.x release. It will also be cherry-picked back to versions 8.2.0 and 7.6.0.
+
+Users on older versions of Bazel can work around this issue by setting `--experimental_remote_cache_ttl=10000d`, with some caveats.
```

Comment:
I think that we should make it clear that the particular value doesn't matter, it just needs to be longer than the expected maximum lifetime of a warm Bazel server.

Reply:
I tried to keep it simple in the top of the blog post so that folks could immediately grab the key takeaways. I separated the value here to make it a bit more clear, but we should have a proper explanation for the workaround near the end.